docs/TESTS.md

```diff
@@ -1,44 +1,110 @@
-### Creating the project in Xcode
+## Xcode Tutorial
 
-Please see <a href="https://developer.apple.com/library/tvos/documentation/DeveloperTools/Conceptual/testing_with_xcode/chapters/02-quick_start.html#//apple_ref/doc/uid/TP40014132-CH2-SW1" target="_blank">Apple's guide on XCTest</a>
+This guide explains how to create and configure an Xcode project for Exercism.
 
-These steps will use the `bob` exercise as an example. If you are not working on `bob`, replace `bob` with your exercise name in the steps below.
+These instructions are written with the expectation that some readers will be very new to Xcode. Each step is described in excruciating detail. However, with a little experience performing these instructions should only take a minute.
 
```

Comment:
Even if you explain this at the end of the tutorial, I would make here a little introduction of the three files we will use to do our exercises and run the tests, and then use always the same name to each file in the whole tutorial. It happened also with the previous tutorial that each time every file was named differently so it was quite confusing at the end to understand which is which. I wrote those name you used, but can be another ones if you consider!

Something like:
_At the end of this tutorial you will work mainly with three files:
- A **Playground File** (MyPlayground.playground): for drafting code snippets and playing with ideas.
- An **Application Source File** (Gigasecond.swift): This is the file where you have to write your final code for the exercise, and this is the file you will submit to exercism.
- A **Tests Source File** (GigasecondTest.swift): This is the file where are placed the tests provided from exercism. 

_Just follow this tutorial to prepare each of these files and understand how to work with them.__